config/initializers/exception_notification.rb

```diff
@@ -0,0 +1,53 @@
+require 'exception_notification/rails'
+
+
+require 'resque/failure/multiple'
+require 'resque/failure/redis'
+require 'exception_notification/resque'
+
+Resque::Failure::Multiple.classes = [Resque::Failure::Redis, ExceptionNotification::Resque]
+Resque::Failure.backend = Resque::Failure::Multiple
+
+
+ExceptionNotification.configure do |config|
+  # Ignore additional exception types.
+  # ActiveRecord::RecordNotFound, Mongoid::Errors::DocumentNotFound, AbstractController::ActionNotFound and ActionController::RoutingError are already added.
+  # config.ignored_exceptions += %w{ActionView::TemplateError CustomError}
+
+  # Adds a condition to decide when an exception must be ignored or not.
+  # The ignore_if method can be invoked multiple times to add extra conditions.
+  config.ignore_if do |exception, options|
+    not Rails.env.production?
```

Comment:
Maybe `Rails.env.development? || Rails.env.test?` - so that staging (or e.g. a demo or whatever environment someone introduces) does get notifications.

Reply:
should that then not use the production environment?

Reply:
I know having a `RAILS_ENV=staging` as common Rails practice, with a different environment configuration. See [rails guides](http://guides.rubyonrails.org/configuring.html#creating-rails-environments).

Reply:
ok, will change it to your suggestion

Reply:
done